artifacts/definitions/MacOS/System/QuarantineEvents.yaml

```diff
@@ -0,0 +1,35 @@
+name: MacOS.System.QuarantineEvents
+description: "This artifact parses the QuarantineEventsV2 database, which provides information
+on when a file was downloaded from the internet."
+
+type: CLIENT
+
+author: Wes Lambert - @therealwlambert
+
+parameters:
+- name: QuarantineGlob
+  default: /Users/*/Library/Preferences/com.apple.LaunchServices.QuarantineEventsV2
+
+precondition:
+      SELECT OS From info() where OS = 'darwin'
+
+sources:
+  - queries:
+     - LET QList = SELECT FullPath FROM glob(globs=QuarantineGlob)
+     - LET QEvents = SELECT * FROM sqlite(file=FullPath, query="SELECT * from LSQuarantineEvent")
+     #  Add delta (978307200 seconds between Cocoa timestamp (2020,1,1) and epoch timestamp (1970,1,1)) to provided Cocoa timestamp
+     - LET QEventsDetails = 
+          SELECT * FROM foreach(
```

Comment:
We probably dont need the foreach here? Just combine QEvents in the same query?

Reply:
I wasn't sure if it would iterate through each one without the `foreach` if there were multiple occurrences of the QEV2 DB.

Reply:
Yeah the sqlite() plugin will emit one row per row returned by sqlite and will build a nested scope with the sql row in it. So the second SELECT .. from scope() is redundant.

It is only not redundant in the plist() case because plist is a function which returns an array so we need foreach to iterate over each item in the array.